openmls/src/messages/proposals.rs

```diff
@@ -339,17 +278,76 @@ impl GroupContextExtensionProposal {
     }
 }
 
-impl ProposalType {
-    /// Check whether a proposal type is supported or not.
-    pub fn is_supported(&self) -> bool {
-        match self {
-            ProposalType::Add
-            | ProposalType::Update
-            | ProposalType::Remove
-            | ProposalType::Presharedkey
-            | ProposalType::Reinit => true,
-            ProposalType::ExternalInit | ProposalType::AppAck => false,
-            ProposalType::GroupContextExtensions => true,
-        }
+// Crate-only types
+
+/// 11.2 Commit
+///
+/// enum {
+///   reserved(0),
+///   proposal(1)
+///   reference(2),
+///   (255)
+/// } ProposalOrRefType;
+///
+/// struct {
+///   ProposalOrRefType type;
+///   select (ProposalOrRef.type) {
+///     case proposal:  Proposal proposal;
+///     case reference: opaque hash<0..255>;
+///   }
+/// } ProposalOrRef;
```

Comment:
Maybe remove these excerpts from the spec? You seem to have removed them everywhere else and I don't think they're very helpful to the reader anyway.

Reply:
I only touched the public docs. I can do that as a follow-up and remove it everywhere in the codebase if you think that makes sense.

Reply:
No worries. We can remove them one by one when we actually touch the corresponding code.